.github/workflows/release.yml

```diff
@@ -151,13 +151,16 @@ jobs:
       - name: Sign archives
         run: |
           echo -e $MINISIGN_KEY > minisign.key
+          echo "$OPENSSL_KEY" > key.pem
           for archive in phylum-*.zip;
           do
             echo $MINISIGN_PASSWORD | minisign -Sm ${archive} -s minisign.key -t 'Phylum - The Software Supply Chain Company'
```

Comment:
This will be phased out over time. If this PR is accepted, I will create an issue to track the eventual removal of `minisign` signatures from our release assets.